### PR TITLE
Master Init but Faster

### LINKED ISSUE #30: MC sleeps for a second at startup
This is to facilitate reconnects, and may not be removable, but even if it is not I still want to document this just so we understand what the cost of everything is.

Old messages apparently.

image

### PR DESCRIPTION
**Pull request by ZephyrTFA**
Cuts the delay for the Master Controller from one second to one deci-second.
References and Closes https://github.com/tgstation/dev-cycles-initiative/issues/30

### CODE REVIEW

**Comment by ZephyrTFA** [comment]
CC @MrStonedOne

**Review by Mothblocks — CHANGES_REQUESTED** [review]
Two things:

1. Should be a define
2. That define should document why we delay at all, which is to give wiggle room for the massive amounts of reconnects at roundstart

**Comment by ZephyrTFA** [comment]
done

**Review by MrsTonedOne — CHANGES_REQUESTED** [review]
You should force push away your first commit because i take MASSIVE issue with the fact you called 1ds a `tick` when a tick is configurable and defaults to 0.5ds on ss13 and even old ss13 defaulted it to 0.9ds

File: code/__DEFINES/MC.dm [hunk 1]
```diff
@@ -1,3 +1,12 @@
+
+/**
+ * The Master Controller sleeps before running initialization logic
+ * allowing clients which were connected from a previous round to establish a connection
+ * and load in before it begins the process of initializing all other subsystems;
+ * thus ensuring that those clients do not drop the connection and need to manually reconnect.
+ */
```
Comment (MrStonedOne) [inline code/__DEFINES/MC.dm:7]: i know mothblocks told you to do this here, but this comment should be in the codedoc for master/Initialize, not here.

**Comment by Mothblocks** [comment]
We don't save the commit descriptions anymore anyway so if you change your PR body to call it a decisecond rather than a tick then the squash commit should be good

**Comment by MrStonedOne** [comment]
> We don't save the commit descriptions anymore anyway so if you change your PR body to call it a decisecond rather than a tick then the squash commit should be good

The current pr is 1 tick (0.5 ds).

the old commit was 1ds (2 ticks).

**Please pay attention and if this pr is merged saying it has 1ds when its 1tick (0.5ds) i will blow a gasket.**

also I'm placing an embargo on @ZephyrTFA working on anything further related to the mc until i am confident they understand the difference between a game tick, a dedisecond, and a subsystem tick.

Knowing the units of time is a prerequisite to working on the MC.

**Comment by ZephyrTFA** [comment]
> snip

I'm terribly sorry you think I pinged you on this PR so that you could fume and get unreasonably angry over something I literally CC'd you on so any mistakes could be caught.

In the future I will make sure that instead of pinging the appropriate people I will simply assume my code works without fault!

**Comment by ZephyrTFA** [comment]
Oh, and make the PR yourself.

**Pull request closed without merging**